### DISCUSSION TITLE
Trying to build Quarkus Native but get Jandex warnings,

### DISCUSSION THREAD

**Discussion by dhoffer** [quarkusio/quarkus#42628] [Q&A]
We are trying to generate native Quarkus apps but we get the following build time warnings:

[WARNING] [io.quarkus.deployment.steps.ReflectiveHierarchyStep] Unable to properly register the hierarchy of the following classes for reflection as they are not in the Jandex index:

However we do have either empty beans.xml file in each dependencies META-INF folder, or a Jandex index file and I have tried adding via the Quarkus application.properties file 'quarkus.index-dependency.[name].group-id & artifact-id' but none of those options resolve the above warning.

We are using mandrel-java17-23.0.5.0-Final

What is the solution to this?

We do have either an empty META-INF/beans.xml file, or a Jandex index file and I have tried adding via the Quarkus application.properties file (we use yaml of same) 'quarkus.index-dependency.[name].group-id & artifact-id' but none of those options resolve the above warning.

[a comment by an automated account is not shown]

**Comment by galderz** [comment]
Does the log show a list of classes? You could look into those? It would seem like something is being registered for reflection access but it's not in the classpath. Maybe there has been dependency changes?
